common/environment/environment.cpp

```diff
@@ -901,11 +999,10 @@ CLocalEnvironment::CLocalEnvironment(const char* environmentFile)
            p.set(root);
    }
 
-   machineCacheBuilt = false;
-   dropZoneCacheBuilt = false;
+   init();
```

Comment:
formatting: should be 4 spaces (I know most of these formatting issues were already there)